h/models/group.py

```diff
@@ -100,6 +100,13 @@ def slug(self):
     def is_public(self):
         return self.readable_by == ReadableBy.world
 
+    @property
+    def logo(self):
```

Comment:
* Does the `logo` belong to the group itself, the group's authority or the as-yet-unspec'ed notion of an "organization?" I have a vague recollection that logos might belong to organizations but that was a while ago...
* What _is_ a logo? A URL?
* How would a logo get associated with a group initially, in this approach?

Reply:
What _is_ a logo? A URL?

Reply:
How does a logo (in this approach) get associated with a group initially?

Reply:
The logo really belongs to the authority/organization- I think those are the same thing now-correct? Right now it's a svg element located in build/images/icons/. For now the publisher/whoever would have to send it to one of the devs and they would add it into the code manually. I think once we build an admin page then uploading a logo could be incorporated into the page. And when we expose that we might need to change the format we accept and where we store it.

Reply:
We should discuss this with other folks, as I don't believe that `authority` and `organization` are the same entity.

I'm not clear on how an icon SVG gets associated with the actual persisted group database entry in the method you're describing here.

Reply:
It doesn't get entered into the db right now-that was my point. Right now it doesn't but in the future once we add the admin page piece it should change so that it does get entered into the db.

Reply:
"For now the publisher/whoever would have to send it to one of the devs and they would add it into the code manually". As I understand it we accept only SVG and no other format. It might be better if I reach out to the design team, request the logo in the correct format or convert the one they send me to SVG and then I create an issue to import the logo into the db, attaching the SVG file.